README.md

```diff
@@ -23,34 +23,34 @@ You can deploy the application to Kubernetes using the Linkerd 2.0 service mesh.
 
 1. Install the `linkerd` CLI
 
-    ```bash
-    curl https://run.linkerd.io/install | sh
-    ```
+```bash
+curl https://run.linkerd.io/install | sh
+```
```

Comment:
As far as I can tell, the indented code block renders properly as long as it's nested inside a list element, as this one is. As it was previously written, it renders like this:

![image](https://user-images.githubusercontent.com/9226/62233103-b249ac00-b37c-11e9-9b4b-8f88a686d2e5.png)

Whereas with your changes, it renders like this:

![image](https://user-images.githubusercontent.com/9226/62233123-c1c8f500-b37c-11e9-8bfb-2d54b9ceafa6.png)

My preference is to keep the previous indentation here, and for all of the code blocks nested inside list elements in the rest of the document.